kong/plugins/zipkin/handler.lua

```diff
@@ -335,6 +286,7 @@ function ZipkinLogHandler:log(conf) -- luacheck: ignore 212
         span:set_tag("http.status_code", try.code)
       end
 
+      tag_with_durations(span)
```

Comment:
Should the balancer span be a child of the proxy span?

My understanding is that it represents the time to assign an upstream from the balancer but not actually attempt the upstream request. Would that ever take more than a few milliseconds? If not, would the number of tries and previous status codes be better represented as tags on the proxy span?

Reply:
1. I think it make sense to put balancer spans as child span of the proxy span.

2. The `balancer_data.tries` represents every request to the upstream, the request can take much longer time if the upstream server is slow to respond or failed to respond, in that case, the way to store these data in spans is more friendly to end-users.